cmd/edgectl/scout.go

```diff
@@ -64,7 +66,14 @@ func NewScout(mode string) (*Scout, error) {
 		return nil, err
 	}
 
-	return &Scout{mode, installID, newInstall}, nil
+	return &Scout{mode, installID, newInstall, ""}, nil
+}
+
+func (s *Scout) SetClusterID(clusterID string) {
+	if s.clusterID != "" {
+		panic(fmt.Sprintf("Trying to replace cluster ID %q with %q", s.clusterID, clusterID))
```

Comment:
`Trying` should not be capitalized
(PS. Sorry I hate making these kind of comments in PRs)